itou/www/companies_views/views.py

```diff
@@ -106,6 +106,23 @@ def report_tally_url(user, company, job_description=None):
     return add_url_params(base_url, params)
 
 
+### Main company view
+
+
+@login_required
+def company_home(request, template_name="companies/company_home.html"):
+    company = get_current_company_or_404(request)
+    if not company.is_active:
+        raise PermissionDenied
+
+    context = {
+        "siae": company,
```

Comment:
Sauf code historique on n'utilise le terme "SIAE" que si on a bien une SIAE (donc faisant de l'IAE) sinon on utilise "company" ;).
```suggestion
        "company": company,
```

Reply:
Yes c'est mieux. 
Par contre je me suis inspiré de ce qui était fait dans la `def show_financial_annexes` et je me demande si ça ne devrait pas être "company" là aussi

Reply:
Non, car toutes les entreprises n'ont pas d'annexes financières, c'est seulement pour les SIAE ;).